spras/config.py

```diff
@@ -230,9 +230,16 @@ def process_config(self, raw_config):
                             try:
                                 obj = [float(obj)]
                             except ValueError:
+                                # Handles arrays and special evaluation types
+                                # TODO: do we want to explicitly bar `eval` if we may use untrusted user inputs later?
```

Comment:
Disallowing `eval` seems like a good idea now that you have this more flexible parsing in place.